lavamoat/browserify/policy-override.json

```diff
@@ -62,37 +40,11 @@
         "crypto": true
       }
     },
-    "ethereumjs-wallet>randombytes": {
```

Comment:
This dependency is still in our policy with the identifier `crypto-browserify>randombytes`. But the auto-generated policy is already granting it access to the `crypto` global in its entirety, so this allowance for the `getRandomValues` method specifically is no longer necessary.